src/cpu/x64/injectors/jit_uni_binary_injector.cpp

```diff
@@ -2596,32 +2592,6 @@ jit_uni_binary_injector_t<isa, Vmm>::execute_cmp_binary(const Vmm &dst,
     pop_opmask(host_, cmp_mask);
 }
 
-template <cpu_isa_t isa, typename Vmm>
-template <typename T>
-typename std::enable_if<std::is_same<T, Xbyak::Zmm>::value
-        || std::is_same<T, Xbyak::Address>::value>::type
-jit_uni_binary_injector_t<isa, Vmm>::execute_prelu_binary(const Vmm &dst, const Vmm &lhs, const T &rhs) const {
-    Vmm dst_vmm = Vmm(dst.getIdx());
-    Xbyak::Opmask maybe_tail_kmask = Xbyak::Opmask(dst.getOpmaskIdx());
-
-    auto aux0_idx = rhs_arg_static_params_.rhs_prelu_helper_vmm_idx;
-    if (static_cast<int>(aux0_idx) == lhs.getIdx()) { 
-        aux0_idx = (aux0_idx + 1) % 16;
-    }
-    const Xbyak::Zmm zmm_aux0 = Xbyak::Zmm(aux0_idx);
-    Xbyak::Opmask aux_kmask = get_aux_kmask();
-
-    push_opmask(host_, aux_kmask);
-    push_vmm(host_, zmm_aux0);
-
-    host_->uni_vpxor(zmm_aux0, zmm_aux0, zmm_aux0);
-    host_->vcmpps(aux_kmask | maybe_tail_kmask, lhs, zmm_aux0, jit_generator::_cmp_lt_os);
-    pop_vmm(host_, zmm_aux0);
-    host_->uni_vmulps(dst_vmm | aux_kmask, lhs, rhs);
-    pop_opmask(host_, aux_kmask);
-}
-
-
```

Comment:
My understanding is the that after this PR, the jit_uni_binary_injector.cpp/hpp would be the same with onednn upstream. Right?

Reply:
The intension was to remove all the custom binary prelu related code, whether there is some extra OV specific code is still unknown.